resources/media_controls.css

```diff
@@ -0,0 +1,38 @@
+button {
+  display: inline-block;
+  width: 24px;
+  height: 24px;
+  border: none;
+}
+
+.controls {
+  display: block;
+  position: fixed;
+  width: 100%;
+  padding: 0 9px;
+}
+
+.hidden {
+  display: none;
+}
+
+.playing {
+  background: url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABIAAAASCAYAAABWzo5XAAAAKklEQVR4AWMYWUABiB2gWAG3OGHQAMT/obgBl/igNGjUoFGDKM8iIwcAAPiWPtn7PkKeAAAAAElFTkSuQmCC") no-repeat;
```

Comment:
Is there a particular reason that PNGs are used for the controls? I imagine that simple SVGs would be better for both maintainability and future compatibility with any given screen density and resolution.

Reply:
There is no support for inline SVG yet. Check https://github.com/servo/servo/pull/23208

Reply:
Makes sense, thanks!